### PR TITLE
[READY]Ports botany belt from para

### PR DESCRIPTION
**Pull request by jughu**
Ports the botany belt from https://github.com/ParadiseSS13/Paradise/pull/1175
You can get the belt from the biogenerator for 300 points

image

:cl: Hyena
add:adds botany belt to the biogenerator for 300 points
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/storage/belt.dm [hunk 1]
```diff
@@ -282,6 +282,24 @@
...
+	    /obj/item/weapon/cultivator,
+	    /obj/item/weapon/shovel/spade,
+	    /obj/item/weapon/reagent_containers/syringe,
+	    /obj/item/weapon/reagent_containers/glass,
+	    /obj/item/weapon/reagent_containers/spray,
+	    /obj/item/seeds,
+	    /obj/item/weapon/hatchet,
+	    /obj/item/weapon/wrench,
+	    /obj/item/device/analyzer/plant_analyzer
+	    )
```
Comment (coiax) [inline code/game/objects/items/weapons/storage/belt.dm]: should also hold lighters, rollies, torches and plant produce?

**Comment by SweaterKittens** [comment]
It wouldn't be too hard to do some minor tweaks and make belts like this look like a little less like recolored toolbelts.

**Comment by jughu** [comment]
@sweaterkittens that might be for another PR if a spriter feels up to it this is porting it from para now

File: code/game/objects/items/weapons/storage/belt.dm (hunk 1 above)
Comment (jughu) [inline code/game/objects/items/weapons/storage/belt.dm]: Yeah i get that on tommorow wasnt fully done yet

**Comment by ExcessiveUseOfCobblestone** [comment]
I may make it look more like a vine one raps around their waist instead of a recoloured toolbelt later in life

File: code/game/objects/items/weapons/storage/belt.dm [hunk 2]
```diff
@@ -282,6 +282,24 @@
...
+	can_hold = list(
+	    /obj/item/weapon/cultivator,
+	    /obj/item/weapon/shovel/spade,
+	    /obj/item/weapon/reagent_containers/syringe,
+	    /obj/item/weapon/reagent_containers/glass,
+	    /obj/item/weapon/reagent_containers/spray,
+	    /obj/item/seeds,
+	    /obj/item/weapon/hatchet,
+	    /obj/item/weapon/wrench,
+	    /obj/item/device/analyzer/plant_analyzer
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/objects/items/weapons/storage/belt.dm]: /obj/item/device/plant_analyzer

**Comment by Armhulen** [comment]
Make a plant that grows belts.

**Comment by ExcessiveUseOfCobblestone** [comment]
make a belt that grows plants.

**Comment by Core0verload** [comment]
Another belt with recolored sprite? Better just extend toolbelt's storage list.

**Comment by SweaterKittens** [comment]
Fair enough, Jughu. And they all serve a different function, Core. It's nice to have job-specific storage. It's better than one item that does everything _coughtrashbagcough_. I'm kind of up to my ears in spriting right now but I'd be willing to take a crack at it once my workload lightens up.

**Comment by dannno** [comment]
>Another belt with recolored sprite? Better just extend toolbelt's storage list.

this tbh

**Comment by jughu** [comment]
should be ready

**Comment by jughu** [comment]
@core0verload no toolbelts are for engineering secbelts are for sec janibelts are for janitor med belts are for medbay BOOKBAG is for librarian im giving botany some love

**Comment by Core0verload** [comment]
toolbelts are for TOOLS, are botany tools less tools than engineering ones?

**Comment by jughu** [comment]
I mean i could add like everything to toolbelts what are tools but would be denied anyways

**Comment by SweaterKittens** [comment]
I think the tough part about fixing belt storage is that while it makes sense for belts to hold any small item, it quickly becomes way too strong. Why would you ever put anything else on your belt if a toolbelt held any small item? So while it's intuitive for toolbelts to be able to hold just about everything, it probably doesn't work from a balance perspective, hence the job-specific belts.

**Comment by Core0verload** [comment]
>I mean i could add like everything to toolbelts what are tools but would be denied anyways

You closed your PR before anyone even had a chance to deny it.

**Comment by Haevacht** [comment]
Man that is fucking ugly.

**Comment by jughu** [comment]
@core0verload if kor dislikes it it pretty tells enough and hey somoene did mentoon swat belts for random clutter 

@haevacht just porting them somoene change the sprites later on anyways so its most likely temporary

**Comment by Core0verload** [comment]
kor dislikes even his own butt, as long as the PR is open, give no fucks

**Comment by jughu** [comment]
@core0verload i mean you pretty much be able to small tiny items in any belt really like sweater said earlier but it be nice if we have job speficic items

I mean in goonstation it would be a utility belt which nearly could hold any job iten mining sec engineering anc such... but this is really for balance so i dont carry things like a full toolbelt with a stetchkin or other things

**Comment by KorPhaeron** [comment]
@tgstation/commit-access 

Is it time to add something to contributing.md about adding more storage slots yet?

**Comment by AnturK** [comment]
If we start to add balance reasons to contributing.dm , it will become 100 page behemoth fast so i'd rather not.

**Comment by KorPhaeron** [comment]
Fair enough

**Comment by coiax** [comment]
Yeah, contributing.dm is only for Nazis and Anti-coiax.

**Comment by lzimann** [comment]
Do we need even more storage powercreep when our backpacks already fit everything you need?

**Comment by calzilla1** [comment]
Yes

**Comment by coiax** [comment]
Other departments get specific toolbelts, why not botany

**Comment by jughu** [comment]
yeah Janitor gets their own belt when they simple can fit their shit in their box backpack and such so why cant botany get one? they even have to get bio points for the generator to get one so they auctally have to work towards one

**Comment by KorPhaeron** [comment]
This is why it's called creep.

Started with toolbelts and now every job has to have a second backpack around their waist

**Comment by ExcessiveUseOfCobblestone** [comment]
so just remove all toolbelts and force people to store their job-important items in their backpack or in things like toolboxes?

**Comment by jughu** [comment]
said toolbelts limited to their job tools its not like they fit a flash in it you call this creep when librarian bag has litteraly 48 slots to put shit in with cut books

**Comment by coiax** [comment]
@KorPhaeron explain why janitors, security, engineering get their own toolbelt, and not botany.

**Comment by KorPhaeron** [comment]
You can fit a flash in the space you just freed up shoving items into your belt though 

@excessiveuseofcobblestone

>why

Because forcing people to make choices about what they want to carry stops them from being one man armies that can perform the duties of every department. 

No reason to interact with (and make yourseld vulnerable to) other people if you can do and carry everything yourself

No being caught off guard by bad situations or tough survival scenarios if you can comfortably carry gear to prepare for everything and enough of it that you'll never run out

**Comment by PKPenguin321** [comment]
>explain why janitors, security, engineering should get their own toolbelt, and not botany.

Janitors don't (they did before we had scaling backpack space (strictly set number of items in your bag regardless of their size) though), security doesn't (but I dare you to try to remove it with sec mains going ballistic on you), engineers probably do kind of need it because they so frequently need quick access to all 7(?) of their tools but the tools could potentially just be moved to a fancy box in their backpack if the need be.

**Comment by KorPhaeron** [comment]
@coiax

I don't think those jobs really need them either.

Toolbelts were more or less made obsolete when we changed backpacks to let you access boxes inside them and hold more small items

**Comment by jughu** [comment]
maybe you should like remove every jobs starting belt and let botany make them as balance uh effort for da belts

**Comment by jughu** [comment]
@korphaeron what if we remove all roundstart belts with exception from engineering/medical and boost the price up from belts from 300 to 500 in the biogenerator so people are auctally forced to use public garden/cooperating with botany for belts

**Comment by ExcessiveUseOfCobblestone** [comment]
Low effort plant making for double the storage capacity doesn't seem very sensible

**Comment by Iamgoofball** [comment]
# this kills tactical plants

**Comment by jughu** [comment]
Well its atleast something then just free storage they auctally need to work for it theres exactly enough biomass in the public garden to create exactly 1 belt of any choose

**Comment by coiax** [comment]
Please either stop complaining about this PR or make a PR to remove toolbelts.

**Comment by jughu** [comment]
@korphaeron so if i see your view of storage creep right

Job belts>hold 7 related items to their job
Bookbag > holds 7 bibles with 7 storage slots>putting them in a book bag creating 48 storage slots on your belt is fine?

You can make said bookbags by using 4 cloth.
You can cut up bedsheets into 3 cloth with wirecutters.
2 bedsheets = 1bookbag>48 storage slots

These belts atleast take effort to make more of then a bookbag which you litteraly only need to cut up 2 bedsheets for and 7 books

**Comment by KorPhaeron** [comment]
I didn't add or defend bookbags try again

**Comment by jughu** [comment]
@korphaeron but still its 48 slots storage on your belt if people really wanted to be a storage creep they make bookbags then using the belts

**Comment by PKPenguin321** [comment]
>Please either stop complaining about this PR or make a PR to remove toolbelts.

Using the existence of questionable features (which are already so ingrained in the playerbase that removing them would be met with a group of people wanting to crucify you) to justify more questionable features. I'm like 99% sure that is the actual literal textbook definition of powercreep

**Comment by Core0verload** [comment]
i'm fucking tired of idiots turning everything into balance issues

**Comment by jughu** [comment]
if people really complain about botany belt being storage creep then might remove the BOOKBAG firstly

**Comment by PKPenguin321** [comment]
Then go remove the book bag then

**Comment by optimumtact** [comment]
the maintainers are clear on storage creep, not acceptable at this time.

**Pull request closed without merging**